src/bucket/BucketController.ts

```diff
@@ -167,6 +183,32 @@ export class BucketController {
         // store folders and files in alphabetical order
         this.folders = contents.folders.sort(compareObjectNames);
         this.files = contents.files.sort(compareObjectNames);
+
+        this.backend.getDeletedContents(this.bucket, this.currentFolder)
+        .then((contents: IFolderContents) => {
```

Comment:
This formatting is not consistent with the rest of the project. The `.then` and `.catch` lines need to be indented one level past the top-level `this.backend` line.